### PR TITLE
Remove old service keys in statistics.json

### PR DESCRIPTION
**Pull request by Flaburgan**
Close https://github.com/diaspora/diaspora/issues/5339

### CODE REVIEW

File: app/presenters/statistics_presenter.rb [hunk 1]
```diff
@@ -8,8 +8,6 @@ def as_json options={}
     base_data.merge(user_counts)
              .merge(post_counts)
              .merge(comment_counts)
-             .merge(all_services)
```
Comment (Flaburgan) [inline app/presenters/statistics_presenter.rb]: Data are already included in base_data, this was useless.

File: app/presenters/statistics_presenter.rb [hunk 2]
```diff
@@ -95,28 +93,10 @@ def local_comments
...
-    result
-  end
-
-  def all_services
-    @all_services ||= all_services_helper
-  end
-
   def available_services
     Configuration::KNOWN_SERVICES.select {|service|
       AppConfig.show_service?(service, nil)
```
Comment (Flaburgan) [inline app/presenters/statistics_presenter.rb]: I don't get the test at https://github.com/diaspora/diaspora/blob/develop/lib/configuration_methods.rb#L41 with "authorized" etc. We just need to know if the service is enabled or not.

**Comment by Flaburgan** [comment]
@jaywink is http://the-federation.info updated to check the new syntax? Maybe you want to still support both syntaxes for pod which are still running 0.4.

@dmorley same remark for http://podupti.me

File: app/presenters/statistics_presenter.rb (hunk 2 above)
Comment (jhass) [inline app/presenters/statistics_presenter.rb]: #5985

Comment (Flaburgan) [inline app/presenters/statistics_presenter.rb]: Oh right! Thank you :)

**Comment by jaywink** [comment]
@Flaburgan not up to date yet, will need to update asap once this is merged in :)

Note, however merges this is - as agreed earlier, this is 0.6 stuff, not a minor release.

File: app/presenters/statistics_presenter.rb (hunk 1 above)
Comment (jaywink) [inline app/presenters/statistics_presenter.rb]: :+1:

**Comment by jaywink** [comment]
Hmm, might as well merge now - thanks!

@dmorley - pods running on develop will not show with correct services after they pull this in so time to update the code if you haven't already. I'd suggest supporting the old services keys until forever for old pods, friendica and redmatrix.

**Pull request merged**